### PR TITLE
Milestone4 logic

### PR DESCRIPTION
**Pull request by BigWizzo**
### React & Redux: Milestone 4 - Logic
- In this milestone, I Implemented the calculate.js module.
- Implemented the operate.js module.
- Added calculate.js to the App component.
- Disabled the Eslint no-unused-vars for `calculate` from `calculate.js`

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      total = (firstNum / secondNum);
+      break;
+    case '*':
+      total = (firstNum * secondNum);
+      break;
+    case '-':
+      total = (firstNum - secondNum);
+      break;
+    case '+':
+      total = (firstNum + secondNum);
```
Comment (bellom) [inline src/logic/operate.js]: Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case subtraction will be -> `total.minus(next)`

OR

http://mikemcl.github.io/big.js/#mod

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const calculate = ({ total, next, operation }, buttonName) => {
+  if (buttonName === '+/-') {
+    const negTotal = (total) * -1;
+    const negNext = (next) * -1;
+    return { total: negTotal, next: negNext };
+  }
+  if (buttonName === '%' || buttonName === '÷' || buttonName === 'X' || buttonName === '-' || buttonName === '+') {
+    const totalResult = operate(total, next, operation);
+    return { total: totalResult.toString() };
+  }
```
Comment (bellom) [inline src/logic/calculate.js:18]: ### Heads-Up:
**we need to add more cases;** 
1. if the `buttonName` = `A/C`, we should return `0`
2. if the `buttonName`  = `.` we should return `0.`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @BigWizzo  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**